runtime/rococo/src/lib.rs

```diff
@@ -1625,6 +1621,7 @@ pub mod migrations {
 
 	/// Unreleased migrations. Add new ones here:
 	pub type Unreleased = (
+		assigned_slots::migration::v1::MigrateToV1<Runtime>,
```

Comment:
It just released, so we should merge https://github.com/paritytech/polkadot/pull/7358 first.